public/index.html

```diff
@@ -50,21 +50,10 @@
         <script src="/lib/autofill-directive/autofill-directive.js"></script>
         <script src="/lib/underscore/underscore-min.js"></script>
         
-        <script src="/modules/app.js"></script>
-        <script src="/modules/base/controllers/main.js"></script>
-        <script src="/modules/base/controllers/login.js"></script>
-        <script src="/modules/base/controllers/navbar.js"></script>
-        <script src="/modules/base/controllers/signup.js"></script>
-
-        <script src="/modules/base/services/User.js"></script>
-        <script src="/modules/base/services/authService.js"></script>
-        <script src="/modules/base/services/menus.js"></script>
-        <script src="modules/base/directives/confirmMessage.js"></script>
-        <script src="modules/base/services/authService.js"></script>
-        <script src="modules/base/services/menus.js"></script>
-        <script src="modules/base/interceptors/authInterceptor.js"></script>
-        <!-- //===== mean-cli hook =====// -->
-
+        <!-- App scripts -->
+        <script src="/dist/scripts/all.min.js"></script>
```

Comment:
not sure, I think we should work by default without the minification, in dev mode, the minification should happen when explicitly want to build using something like: `grunt prod`